### DISCUSSION TITLE
Escape disallowed tags & don't auto generate closing tag

### DISCUSSION THREAD

**Discussion by augerT** [apostrophecms/sanitize-html#543] [Q&A]
Is it possible to do the titled description?

I have a large chunk of HTML being converted from an RTF file that may contain angled brackets. Running this HTML through the sanitizer seems to auto-generate closing tags if angled brackets exist. 

I.e. I have HTML `<p>This is a <test></p>`

Sanitizing this with disallowedTagsMode on 'escape' produces : `<p>This is a &lt;test&gt;&lt;/test&gt;</p>`

But what I want is `<p>This is a &lt;test&gt;</p>`

I've read through the documentation and I don't see anything on auto-generating close tags.. wondering if this is possible or if there's some sort of workaround. 

Thanks!

**Comment by boutell** [comment]
Take a look at the `selfClosing` option.
